src/Text/Pandoc/Filter/IncludeCode.hs

```diff
@@ -189,26 +207,48 @@ splitLines = return . Text.lines
 joinLines :: Lines -> Inclusion Text
 joinLines = return . Text.unlines
 
+includeLink :: Inclusion ()
+includeLink =  do
+    links <- asks include 
+    bases <- asks base 
+    setLink bases links 
+
 allSteps :: Inclusion Text
 allSteps =
-  readIncluded >>= splitLines >>= includeByMode >>= dedentLines >>= joinLines
-
-includeCode' :: Block -> IO (Either InclusionError Block)
-includeCode' cb@(CodeBlock (id', classes, attrs) _) =
-  case parseInclusion (HM.fromList attrs) of
-    Right (Just spec) ->
-      runInclusion' spec allSteps >>= \case
-        Left err -> return (Left err)
-        Right (contents, state) ->
-          return
-            (Right
-               (CodeBlock
-                  (id', classes, modifyAttributes state classes attrs)
-                  contents))
-    Right Nothing -> return (Right cb)
-    Left err -> return (Left err)
-includeCode' x = return (Right x)
+    includeLink >> readIncluded >>= splitLines >>= includeByMode >>= dedentLines >>= joinLines
+
+getName :: Text -> Text
+getName filepath = last $ splitOn (Text.pack "/") filepath
```

Comment:
`"/"` is not always the path separator (especially on Windows!). Consider using the appropriate `splitFileName` function from the `filepath` package.